string group_name
trajectory_msgs/JointTrajectory joint_trajectory

#goal definition
duration time_offset
string target_link
---
trajectory_msgs/JointTrajectory actual_trajectory
motion_planning_msgs/ArmNavigationErrorCodes error_code
---
#HeadMonitorStatus status
#motion_planning_msgs/RobotState paused_state
#mapping_msgs/CollisionMap paused_collision_map

